src/app/learn/get-started/advanced/sticky-settings/integrations/custom-components/index.html

```diff
@@ -0,0 +1,52 @@
+<stache
+  pageTitle="Custom Components"
+  navTitle="Custom components"
+  navOrder="20">
+
+  <p>
+    To save user configuration settings in custom components, use the <sky-code>SkyUIConfigService</sky-code> as illustrated in the following code sample. The <sky-code>key</sky-code> that you provide to the <sky-code>SkyAppUIConfigService</sky-code> must be unique within your SPA.
+  </p>
+
```

Comment:
I think this needs the same warning as the SKY UX components pages... "For Blackbaud developers, the settingsKey must be added to a server-side whitelist. Blackbaud developers can learn how to add keys to that whitelist on the UI Config Service website."

Reply:
Good catch. Added this locally and will push the change shortly.